wagtail/project_template/home/app.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class HomeConfig(AppConfig):
+    default_auto_field = "django.db.models.BigAutoField"
```

Comment:
nitpick: `AutoField` is more appropriate for this app

Reply:
@zerolab Is it though? Since Django 3.2, Django has `default_auto_field` set to BigAutoField on all newly-created projects and apps. I think the only reason Django doesn't use this as the overall default is backwards compatibility.

Reply:
Let's keep as is, to keep in sync with Django.